packages/app/scripts/start.js

```diff
@@ -337,13 +337,14 @@ function run(port) {
     });
     http
       .createServer(function (req, res) {
-        if (req.url.includes('.js')) {
-          proxy.web(req, res, { target: 'http://localhost:3000' });
-        } else {
+        // Not sure if this is always "/"?
+        if (req.url === '/') {
           proxy.web(req, res, {
             target: 'http://localhost:3000/frame.html',
             ignorePath: true,
           });
+        } else {
+          proxy.web(req, res, { target: 'http://localhost:3000', ws: true });
```

Comment:
This still returns in: `WebSocket connection to 'ws://localhost:3002/sockjs-node/062/01tbrbdx/websocket' failed: `

Not sure if anyone has a clue where this should be pointing, if the url is correct I can look further into how to make this work properly